plugins/SitesManager/Controller.php

```diff
@@ -371,7 +371,7 @@ private function getCmsInstruction()
 
         $detectedCms = $this->siteContentDetector->getSiteContentDetectionById(reset($detectedCMSes));
 
-        if (null === $detectedCms) {
+        if (null === $detectedCms || empty($detectedCms::getInstructionUrl())) {
```

Comment:
Could that potentially hide available instructions, if multiple CMSs are detected, and the first one happens to be Cloudflare?

For example Cloudflare+Drupal. It should be possible to step over the Cloudflare detection and display the next possible candidate if one is available.

Reply:
good point. will check that.